### ISSUE TITLE
How do I add a textview on the alert dialog?

### ISSUE THREAD

**Issue by mooner1022** [Shashank02051997/FancyAlertDialog-Android#6]
Hello,
I want to attatch a textview on the dialog but I think there isn't an option to do that. would you please tell me the way to add it or add the function to it? Thanks for the fancy alert dialog.

**Comment by Shashank02051997** [comment]
I am not understanding what's your point.

**Comment by mooner1022** [comment]
In other dialogs(like the original android dialog), it has an option to add a textview in the dialog. But in this 'Fancy Dialog', I can't add a textview in the dialog. Even though I love this beautiful dialog, I can't use it because of that problem.

**Comment by shrirampanchal** [comment]
> I am not understanding what's your point.

Yes, same demand is here as like  mooner1022 said  , if I want to add a view in the middle message area (I want to add my custom layout in it) then it should capable of adding any type of view (so that anyone can make their own custom layout and attach it) please make these changes and it will be awesome to use.  And thanks for making this excellent work.
